mart/callbacks/progress_bar.py

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2022 Intel Corporation
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+import tqdm
+from pytorch_lightning.callbacks import TQDMProgressBar
+
+__all__ = ["ProgressBar"]
+
+
+class ProgressBar(TQDMProgressBar):
+    """Display progress bar of attack iterations with the gain value."""
+
+    def init_train_tqdm(self):
+        bar = super().init_train_tqdm()
+        bar.leave = False
+        bar.set_description("Attack")
```

Comment:
It looks like the progress bar is not changed.
```console
Epoch 0:  10%|██▉                | 20/200.0 [00:00<00:06, 26.93iter/s, loss=40.6]
```

Reply:
I never got the progress bar working for the attack.

Reply:
Resolved in #153